package.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "dash-table",
-  "version": "3.1.4",
+  "version": "3.1.5",
   "description": "Dash table",
-  "main": "lib/bundle.js",
+  "main": "dash_table/bundle.js",
   "scripts": {
     "preprivate::opentests": "run-s private::wait*",
     "preprivate::runtests": "run-s private::wait*",
     "private::build": "node --max_old_space_size=4096 node_modules/webpack/bin/webpack --display-reasons --bail",
-    "postprivate::build": "cp -R lib/* dash_table",
+    "private::build:js": "run-s \"private::build -- --mode production\"",
+    "private::build:js-dev": "run-s \"private::build -- --mode development\"",
+    "private::build:js-test": "run-s \"private::build -- --config webpack.test.config.js\"",
+    "private::build:js-test-watch": "run-s \"private::build -- --config webpack.test.config.js --watch\"",
+    "private::build:py": "./extract-meta src/dash-table/DataTable.js > dash_table/metadata.json && cp package.json dash_table/bundle.json",
```

Comment:
Apart from going back to building in /dash_table, this is the major change, when copying package.json, rename it to something else